roles/neutron-data-network/tasks/main.yml

```diff
@@ -25,6 +25,18 @@
     name: iputils
   when: os == 'rhel'
 
+- name: install custom ucarp scripts - rhel
+  template:
+    src: "usr/local/libexec/{{ item }}"
```

Comment:
Yep, all it adds is a `label` argument to `ip addr add/del` so it shows up in `ip addr show` with `:ucarp` as it does on Ubuntu. (Using a label with delete is a no-op; just there for symmetry)